app/(login)/_layout.tsx

```diff
@@ -0,0 +1,5 @@
+import { Stack } from 'expo-router';
+
+export default function RootLayout() {
+  return <Stack screenOptions={{ headerShown: false }}></Stack>;
```

Comment:
No routes inside this Stack? 

Also `RootLayout` might not be an appropriate name here.

Reply:
removed this. Not needed